### PR TITLE
Metrics Glossary: IoU images

### PR DESCRIPTION
**Pull request by y27choi**
### Linked issue(s):
[KOL-2602](https://linear.app/kolena/issue/KOL-2602/coordinates-for-iou-example-look-like-they-correspond-to-the-wrong), [KOL-2610](https://linear.app/kolena/issue/KOL-2610/iou-box-image-and-latex)

### What change does this PR introduce and why?
* use consistent formatting and better place the coordinates at the bottom right

### Please check if the PR fulfills these requirements

- [ ] Include reference to internal ticket and/or GitHub issue "Fixes #NNNN" (if applicable)
- [ ] Relevant tests for the changes have been added
- [ ] Relevant docs have been added / updated

### CODE REVIEW

**Comment by y27choi** [comment]
Waiting for [PR#113](https://github.com/kolenaIO/kolena/pull/113)

File: docs/assets/images/metrics-iou-2dbbox-light.svg
Comment (munkyshi) [inline docs/assets/images/metrics-iou-2dbbox-light.svg:1]: It feels like `(x_a2, y_a2)` is not in the corner for the red box, but instead the point where the red box and blue box intersect.

Comment (munkyshi) [reply docs/assets/images/metrics-iou-2dbbox-light.svg:1]: Pinged Phoebe directly to avoid the game of telephone.

**Comment by gordonhart** [comment]
Would benefit from a rebase to pick up changes on trunk and reduce visible diff.

File: docs/assets/images/metrics-iou-2dbbox-light.svg
Comment (y27choi) [reply docs/assets/images/metrics-iou-2dbbox-light.svg:1]: Thanks Andrew. I got the new images from Phoebe and PR is updated now.

**Review by munkyshi — APPROVED** [review]

**Pull request merged**